bin/bump

```diff
@@ -22,6 +23,7 @@ OptionParser.new do |opts|
   opts.on("-m", "--commit-message MSG", String, "Append MSG to the commit message.") { |msg| options[:commit_message] = msg }
   opts.on("--no-bundle", "Do not bundle.") { options[:bundle] = false }
   opts.on("--tag", "Create git tag from version (only if commit is true).") { options[:tag] = true }
+  opts.on("--tag-prefix TAG_PREFIX", 'Shellwords', "Prefix the tag with this string, ex. 'v'") { |tag_prefix| options[:tag_prefix] = tag_prefix }
```

Comment:
idk why this says shellword and requires shellworks ...
```suggestion
  opts.on("--tag-prefix TAG_PREFIX", String, "Prefix the tag, default: 'v'") { |tag_prefix| options[:tag_prefix] = tag_prefix }
```

Reply:
I had to do this because `String` type is unable to parse `""`:
```Failures:

  1) Bump git should tag the version without any prefix if --tag flag given and --tag-prefix is ''
     Failure/Error: raise "FAILED #{cmd} --> #{result}" if $?.success? != !options[:fail]

     RuntimeError:
       FAILED bump/bin/bump patch --tag --tag-prefix "" --> bump/bin/bump:28:in `<main>': invalid argument: --tag-prefix  (OptionParser::InvalidArgument)
     # ./spec/spec_helper.rb:18:in `run'
     # ./spec/bump_spec.rb:550:in `bump'
     # ./spec/bump_spec.rb:104:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:31:in `block (2 levels) in inside_of_folder'
     # ./spec/spec_helper.rb:29:in `chdir'
     # ./spec/spec_helper.rb:29:in `block in inside_of_folder'
```

Reply:
this works:
```suggestion
  opts.on("--tag-prefix TAG_PREFIX", "Prefix the tag with this string, ex. 'v'") { |tag_prefix| options[:tag_prefix] = tag_prefix }
```